index.html

```diff
@@ -34,6 +34,33 @@ <h2 class="toolBar">My Logo</h2>
         </section>
 
         <section class="works">
+            <div class="detail">
+                <span class="cross">&times;</span>
+                    <h1 id="projectName"></h1>
+                <div class="list">
+                    <span>CANOPY</span>
+                    <ul>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+         
+                <img src="" alt="" id="pjImg">
+             
+                <div class="status">
+                    <p id="paragraph"></p>
+
+                    <div class="language">   
+                    <ul class="techno"> 
+                    </ul>
+
+                    <div class="switch">
+                        <button class="liveBtn">See Live <img src="./img/liveIcon.svg" alt=""> </button>
+                        <button class="sourceBtn">See Source <img src="./img/gitFrame.png" alt=""></button>
+                    </div>     
```

Comment:
- [ ] Your detail popup window `see live` and `see source` button is not working when I click the button that why you should add the actual source code and demo link for that buttons .If you don't have another project you can add portfolio demo link and source code link for all the project as temporarily.